### DISCUSSION TITLE
Champions!

### DISCUSSION THREAD

**Discussion by oscarlevin** [OpenMathBooks/small-documents#10] [General]
We need individuals to take on the role of **champion** for various improvements we have discussed at the workshop.  Some such projects include,

- Documentation: Jason
   - Reference: Jason (Aaron, Sean, Nathan, Chrissi, Duane, Oscar)
   - How Tos: Hammad
   - Landing page: Chrissi (Oscar, Jen)
   - Guide improvements
- Jupyter Notebooks: Sean (Sean)
- Slides: Aaron
- Syllabus: Jen
- Worksheet/workbook: Mitch
- Bibliography: David F.
- Journal Styles: Oscar & Hammad
- New CSS/Curb Appeal: Jason & Jen & Hammad

**Comment by siefkenj** [comment]
We're making incremental progress on documentation with 36 new tags documented :-)
